src/main/java/com/epam/admissions_committee/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.admissions_committee.controller;
+
+import com.epam.admissions_committee.controller.dto.UserDto;
+import com.epam.admissions_committee.service.UserService;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Log4j2
+public class UserController {
+    private final UserService userService;
+
+    @ApiOperation("Get users")
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/user")
```

Comment:
I would recommend to use @RequestMapping("/user") instead of duplicate the same url in each method